index.html

```diff
@@ -9,6 +9,7 @@
 <body>
     <ul>
         <li>This is my feature 1</li>
+        <li>This is my feature 2</li>
```

Comment:
Looks like crap. Try again.

Reply:
You don't know what you're talking about. It's perfect. PR approved.